### PR TITLE
addon installer: remove referer code

### PR DESCRIPTION
**Pull request by tamland**
Doesn't appear to be used for anything.

There is one change here: the language migration will now bypass master pin. This didn't make any sense to me as it is a kind of update (which are allowed without pin) and it's required update.

cc @Montellese @notspiff

### CODE REVIEW

File: xbmc/utils/RssManager.cpp [hunk 1]
```diff
@@ -73,9 +73,8 @@ void CRssManager::OnSettingAction(const CSetting *setting)
     ADDON::CAddonMgr::GetInstance().GetAddon("script.rss.editor",addon);
     if (!addon)
     {
-      if (!CGUIDialogYesNo::ShowAndGetInput(CVariant{24076}, CVariant{24100}, CVariant{"RSS Editor"}, CVariant{24101}))
+      if (!CAddonInstaller::GetInstance().InstallModal("script.rss.editor", addon))
```
Comment (MartijnKaijser) [inline xbmc/utils/RssManager.cpp]: are those strings obsolete?

Comment (tamland) [inline xbmc/utils/RssManager.cpp]: nope. used in InstallModal

**Comment by akva2** [comment]
It is used by the mirrors.

**Comment by tamland** [comment]
> It is used by the mirrors.

For what?

**Comment by akva2** [comment]
in order to try to estimate the number of installs. a blank referer = new install, non-blank = updated from this version.

**Comment by MartijnKaijser** [comment]
@notspiff afaik "we" currently do not use this (any more).

**Comment by tamland** [comment]
Yes, precisely: just wanted to make sure there weren't any other hidden uses I don't know about.

**Comment by notspiff** [comment]
that's the only thing it was ever used for.

**Comment by Memphiz** [comment]
thx for looking into it

**Comment by tamland** [comment]
@notspiff ok thanks!
@Memphiz If you're referring to the other commit I think you're just not reading timestamps correctly today.. but thanks anyway:)

**Comment by tamland** [comment]
jenkins build this please

**Comment by Memphiz** [comment]
@tamland nope my comment is in the right place ... *wonders*

**Comment by tamland** [comment]
carry on then..
jenkins build this please

**Pull request merged**